user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+.. _user-guide-intro-log-in-edit-profile:
+
+Log In, View Your User Page, and Set Up Your Profile
+====================================================
+
+To get to your OroCRM account, enter the OroCRM instance url.
+
+.. image:: ./img/intro/login.png
+
+.. hint::
+
+    The *"login using google"* option may not be available for your organization, check with your OroCRM Administrator.
```

Comment:
Edit: The *"login using google"* option may not be available for your organization, so check with your OroCRM Administrator.